mmdet/apis/ote/apis/detection/openvino_task.py

```diff
@@ -41,140 +47,72 @@
-    output = {
-        'labels': get_output(net, outs, 'labels'),
-        'boxes': get_output(net, outs, 'boxes')
-    }
-    valid_detections_mask = output['labels'] >= 0
-    output['labels'] = output['labels'][valid_detections_mask]
-    output['boxes'] = output['boxes'][valid_detections_mask]
-    output['boxes'][:, :4] /= np.tile(input_size, 2)[None]
-    return output
-
-
-class OpenVINODetectionInferencer(BaseOpenVINOInferencer):
+class OpenVINODetectionInferencer(BaseInferencer):
     def __init__(
         self,
+        hparams: OTEDetectionConfig,
         labels: List[LabelEntity],
         model_file: Union[str, bytes],
         weight_file: Union[str, bytes, None] = None,
-        confidence_threshold: float = 0.0,
         device: str = "CPU",
         num_requests: int = 1,
     ):
         """
         Inferencer implementation for OTEDetection using OpenVINO backend.
 
+        :param hparams: Hyper parameters that the model should use.
         :param labels: List of labels that was used during model training.
         :param model_file: Path OpenVINO IR model definition file.
         :param weight_file: Path OpenVINO IR model weights file.
-        :param confidence_threshold: Confidence threshold for passing detection to the output.
         :param device: Device to run inference on, such as CPU, GPU or MYRIAD. Defaults to "CPU".
         :param num_requests: Maximum number of requests that the inferencer can make. Defaults to 1.
 
         """
-        super().__init__(model_file, weight_file, device, num_requests)
         self.labels = labels
-        self.input_blob_name = 'image'
-        self.n, self.c, self.h, self.w = self.net.input_info[self.input_blob_name].tensor_desc.dims
-        self.keep_aspect_ratio_resize = False
-        self.pad_value = 0
-        self.confidence_threshold = confidence_threshold
-
-    @staticmethod
-    def resize_image(image: np.ndarray, size: Tuple[int], keep_aspect_ratio: bool = False) -> np.ndarray:
-        if not keep_aspect_ratio:
-            resized_frame = cv2.resize(image, size)
-        else:
-            h, w = image.shape[:2]
-            scale = min(size[1] / h, size[0] / w)
-            resized_frame = cv2.resize(image, None, fx=scale, fy=scale)
-        return resized_frame
+        try:
+            model_adapter = OpenvinoAdapter(create_core(), model_file, weight_file, device=device, max_num_requests=num_requests)
+            label_names = [label.name for label in self.labels]
+            self.configuration = {**attr.asdict(hparams.inference_parameters.postprocessing,
+                                  filter=lambda attr, value: attr.name not in ['header', 'description', 'type', 'visible_in_ui']),
+                                  'labels': label_names}
+            self.model = Model.create_model(hparams.inference_parameters.class_name.value, model_adapter, self.configuration)
+            self.model.load()
+        except ValueError as e:
+            print(e)
```

Comment:
What error is it for? And why is it safe to ignore it?

Reply:
This block check that model can be found in ModelAPI models or model_wrappers of task. Now ModelAPI doing it itself and I can delete `try except`